pos_no_bank_statement/model/point_of_sale.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# This file is part of OpenERP. The COPYRIGHT file at the top level of
+# this module contains the full copyright notices and license terms.
+
+from openerp import models, api
+
+
+class POSOrder(models.Model):
+    _inherit = "pos.order"
+
+    @api.v7
```

Comment:
Right, I'm not very experienced with new API. Could you please show how this method should look like in new API so that it would work despite the original method is written in old API?

Cheers